pkg/autoscaler/autoscaler.go

```diff
@@ -326,11 +332,12 @@ var _ = Describe("[Feature:Machines] Autoscaler should", func() {
 		var machineAutoscalers []*caov1beta1.MachineAutoscaler
 
 		By(fmt.Sprintf("Creating %v machineautoscalers", len(machineSets)))
+		transientMachineSetsMinSize := int32(1)
 		var clusterExpansionSize int
 		for i := range machineSets {
 			clusterExpansionSize++
-			glog.Infof("Create MachineAutoscaler backed by MachineSet %s/%s - min:%v, max:%v", machineSets[i].Namespace, machineSets[i].Name, 1, 2)
-			asr := machineAutoscalerResource(machineSets[i], 1, 2)
+			glog.Infof("Create MachineAutoscaler backed by MachineSet %s/%s - min:%v, max:%v", machineSets[i].Namespace, machineSets[i].Name, transientMachineSetsMinSize, 2)
```

Comment:
If this were a `By`, it would auto be slurped by the GinkgoWriter, so it would only display for a particular test if it failed, WDYT?

Reply:
let's talk about this, part of the reason i backed off on using `By` for these inner loop logs is that when they are printed in the logs it looks like the test is starting a new "step" everytime it does this. is there something similar to `By` that i can use more like a log emit?